src/plugins/imagePlugin.tsx

```diff
@@ -289,7 +290,8 @@ export default function createImagePlugin(
     }));
 
     const copyString = getCopyString(title, imageUrl, options.path, copyright, locale);
-    const figureId = `figure-${id}`;
+    const unique = uniqueId();
```

Comment:
Vet vi hvordan `uniqueId` holder styr på hva som er unikt?

Reply:
Det er en teller som begynner på 1 og inkrementerer en for kvar gong uniqueId() kalles.